### DISCUSSION TITLE
How to implement custom code block prism color theme?

### DISCUSSION THREAD

**Discussion by digitalniakademie** [facebook/docusaurus#9340] [Q&A]
I have tried to create gdscript color theme for code blocks. I have duplicated prism theme into a new folder  in \node_modules\prism-react-renderer\themes and created/edited the colors. Then I have added the code into docusaurus.config.js under Themeconfig:

`prism: {
      theme: godotdarkCodeTheme,
      darkTheme: godotdarkCodeTheme,
      additionalLanguages: ['gdscript', 'glsl'],
    },`

I had  this constant defined at the beginning:
`const godotdarkCodeTheme = require('prism-react-renderer/themes/godotDark'); 
`
Everything worked fine locally, but it does not compile on the server. What is the correct way to implement custom themes for code block? Where to put those files?

**Comment by slorber** [comment]
> Everything worked fine locally, but it does not compile on the server. 

It's impossible to help you with a repro. You don't even share the compilation error you got. 

If you want help, help us help you by providing all the relevant infos and save us time by providing a repro.
